Description: fixes overzealous input checking
 Netmask /0 is perfectly fine, with this patch it's possible to use it.
Author: Victor Engmark <victor.engmark@gmail.com>
Bug-Debian: http://bugs.debian.org/332468
Origin: other, https://github.com/l0b0/ipcalc/commit/9388977d5634256335e6d217546a39f3d9c25707
Reviewed-By: Bartosz Fenski <fenio@debian.org>

--- ipcalc-0.41.orig/contributors
+++ ipcalc-0.41/contributors
@@ -15,3 +15,4 @@ Steve Kent
 Sven Anderson       
 Torgen Foertsch
 Tim Brown
+Victor Engmark
--- ipcalc-0.41.orig/ipcalc
+++ ipcalc-0.41/ipcalc
@@ -981,7 +981,7 @@ sub argton
    # bit-count-mask (24 or /24)
    $arg =~ s/^\/(\d+)$/$1/;
    if ($arg =~ /^\d{1,2}$/) {
-      if ($arg < 1 || $arg > 32) {
+      if ($arg < 0 || $arg > 32) {
          return -1;
       }
       for ($i=0;$i<$arg;$i++) {
